packages/landing/src/components/debug.tsx

```diff
@@ -158,24 +157,22 @@ export class Debug extends Component<{ map: maplibregl.Map }, DebugState> {
     );
   }
 
-  setWMTSLink(): void {
-    const wmtsLink = WindowUrl.toTileUrl({
+  getWMTSLink(): string {
+    return WindowUrl.toTileUrl({
       urlType: MapOptionType.Wmts,
       tileMatrix: Config.map.tileMatrix,
       layerId: Config.map.layerId,
       config: Config.map.config,
-      dateRange: Config.map.dateRange,
+      date: Config.map.filter.date,
```

Comment:
This would be better taking `filter: Config.map.filter` 

although looking at this, I wonder i f we should have `WindowUrl.toConfigTileUrl(Wmts)` or similar

Reply:
That will be much better, which just take a `urlType` and we have all the configs to use directly.

Reply:
I have made the change in #2710